network.go

```diff
@@ -60,6 +62,24 @@ func createHTTPClient(jsonKeyPth string) (*http.Client, error) {
 
 		return shouldRetry, err
 	}
+	retryClient.ResponseLogHook = func(debugLogger retryablehttp.Logger, resp *http.Response) {
+		reqDump, err := httputil.DumpRequestOut(resp.Request, false)
+		if err != nil {
+			debugLogger.Printf("failed to dump request: %v", err)
+		}
+		debugLogger.Printf("Request: %s", reqDump)
+
+		dumpBody := false
+		if resp.StatusCode >= 300 || resp.StatusCode < 200 {
+			dumpBody = true
+		}
+
+		respDump, err := httputil.DumpResponse(resp, dumpBody)
+		if err != nil {
+			debugLogger.Printf("failed to dump response: %s", err)
+		}
+		debugLogger.Printf("Response: %s", respDump)
+	}
```

Comment:
As far as I can see in the code that we are using this retryClient as a parameter for the oath library and the androidpublisher. Did you make a test run with these changes? Just asking because this has the potential of printing some unwanted secret values by dumping the request and response.

Reply:
yes it could have printed Auth header value in debug mode.